query/security/sts/src/main/java/org/codice/ddf/admin/security/sts/actions/GetStsClaimsAction.java

```diff
@@ -15,22 +15,27 @@
 
 import org.codice.ddf.admin.common.actions.GetAction;
 import org.codice.ddf.admin.common.fields.base.ListFieldImpl;
+import org.codice.ddf.admin.configurator.ConfiguratorFactory;
 import org.codice.ddf.admin.security.common.fields.sts.StsClaimField;
+import org.codice.ddf.admin.security.common.services.StsServiceProperties;
 
 public class GetStsClaimsAction extends GetAction<ListFieldImpl<StsClaimField>> {
 
     public static final String NAME = "claims";
 
     public static final String DESCRIPTION = "All currently configured claims the STS supports.";
 
-    public GetStsClaimsAction() {
+    ConfiguratorFactory configuratorFactory;
+
+    public GetStsClaimsAction(ConfiguratorFactory configurator) {
         super(NAME, DESCRIPTION, new ListFieldImpl<>(StsClaimField.class));
+        this.configuratorFactory = configurator;
     }
 
     @Override
-    public ListFieldImpl performAction() {
-        StsClaimField claim = new StsClaimField();
-        claim.setValue("testClaim");
-        return new ListFieldImpl<>(StsClaimField.class).add(claim);
+    public ListFieldImpl<StsClaimField> performAction() {
+        ListFieldImpl<StsClaimField> claims = new ListFieldImpl<>(StsClaimField.class);
+        claims.setValue(new StsServiceProperties().getConfiguredStsClaims(configuratorFactory));
```

Comment:
We should consider avoiding the `setValue` in production. It might be better to use `addAll` instead on the `ListField`